File: Backend/TopSys.TopConWeb.Domain/Entities/Visitas/VisitaContato.cs
namespace TopSys.TopConWeb.Domain.Entities.Visitas
{
    public class VisitaContato
    {
        public int Usina { get; set; }
        public int NumeroVisita { get; set; }
        public int AnoVisita { get; set; }
        public int Sequencia { get; set; }
        public string Nome { get; set; }
        public CadastroGeral Funcao { get; set; }
        public int? FuncaoCodigo { get; set; }
        public int DddTelefone { get; set; }
        public int Telefone { get; set; }
        public int DddCelular { get; set; }
        public int Celular { get; set; }
        public string Email { get; set; }
    }
}

File: Frontend/TopConWeb-AngularV8/src/app/classes/visita/visita-contato.ts
import { CadastroGeral } from "../cadastro-geral/cadastro-geral";

export class VisitaContato {
    usina: number = 0;
    numeroVisita: number = 0;
    anoVisita: number = 0;
    sequencia: number = 0;
    nome: string = '';
    
    funcao: CadastroGeral;
    funcaoCodigo: number = 0;

    dddTelefone: number = 0;
    telefone: number = 0;
    dddCelular: number = 0;
    celular: number = 0;
    email: string = '';
}